airbyte-integrations/connectors/source-metabase/source_metabase/schemas/dataset_query_views.json

```diff
@@ -0,0 +1,53 @@
+{
+  "type": ["null", "object"],
+  "properties": {
+    "viewed_on": {
+      "type": ["null", "string"]
```

Comment:
```suggestion
      "type": "string"
```
As this field is declared as the cursor I don't think we should consider it nullable, wdyt?